README.rst

```diff
@@ -40,3 +40,19 @@ support for an unsupported version of Django.
 **2014-12-10 - 1.1**: Django 1.5 - 1.7
 
 **2013-07-29 - 1.0**: Django 1.5 - 1.6
+
+
+**Roadmap**
```

Comment:
You can put this info in the releases section since this roadmap is changing the release policy for localflavor.
https://github.com/django/django-localflavor/blob/master/docs/index.rst#releases